Dixit_Logic/Classes/GameState.cs

```diff
@@ -49,23 +51,35 @@ class GameState : IGameState
         private IList<IPlayer> _players;
 
         /// <summary>
+        /// Store a logical value wheter the game is started or not.
```

Comment:
I will make this class public too. That way everybody can use GameState type. But I have implemented the whole logic such that nobody could modify the game's state except logic.